src/components/tabs/__internal__/tabs-header/tabs-header.style.ts

```diff
@@ -165,27 +99,89 @@ const StyledVerticalTabsWrapper = styled.div<StyledVerticalTabsWrapperProps>`
   flex-direction: column;
 `;
 
-const StyledTabsBottomBorderWrapper = styled.div`
+type StyledWrapperProps = {
+  align: string;
+  position: string;
+  role?: string;
+  extendedLine?: boolean;
+  noRightBorder?: boolean;
+  isInSidebar?: boolean;
+};
+
+type StyledNavigationButtonWrapperProps = {
+  position: string;
+  visible: boolean;
+};
+
+const StyledWrapper = styled.section<StyledWrapperProps>`
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  ${({ extendedLine = true }) =>
+    !extendedLine &&
+    css`
+      width: fit-content;
+    `}
+
+  ${({ position }) => position === "top" && "white-space: nowrap"};
+
+  ${({ align }) =>
+    align === "right" &&
+    css`
+      justify-content: flex-end;
+      text-align: right;
+    `}
+`;
+
+const StyledNavigationButtonWrapper = styled.div<StyledNavigationButtonWrapperProps>`
+  width: 48px;
+  height: 48px;
+
+  ${({ position, visible }) => css`
+    display: ${visible ? "block" : "none"};
+    ${position === "right" && "right: 0; position: absolute;"}
+  `}
+`;
+
+const StyledNavigationButton = styled.button`
```

Comment:
**issue**: I noticed the navigation buttons don't seem to use Carbon's focus border styling. This may impact accessibility by having inconsistent focus visuals. Could we update this?

![navigation-button-focus-border](https://github.com/user-attachments/assets/1dfcffc1-953d-4014-8639-c873fbd72389)

Reply:
They [shouldn't actually be supported at all](https://zeroheight.com/2ccf2b601/p/54468c-tab/t/page-54468c-69590208-28) according to the DS 😄 I'll sort it out